src/main/java/MainClass.java

```diff
@@ -0,0 +1,56 @@
+public class MainClass {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[(int) (Math.random() * 9 + 1)];
+        getRandomFigure(figures);
+        for (int i = 0; i < figures.length; i++) {
+            System.out.println(figures[i].draw());
+        }
+    }
+
+    public static String getRandomColor() {
+        int random = (int) (Math.random() * 7);
+        switch (random) {
+            case 1:
+                return "Красный";
+            case 2:
+                return "Оранжевый";
+            case 3:
+                return "Желтый";
+            case 4:
+                return "Зеленый";
+            case 5:
+                return "Голубой";
+            case 6:
+                return "Синий";
+            case 7:
+                return "Фиолетовый";
+            default:
+                return "Черный";
+        }
+    }
+
+    public static Figure[] getRandomFigure(Figure[] figures) {
```

Comment:
do you need to return array if you are not create a new array but using the same from parameter